src/cli/info.js

```diff
@@ -1,20 +1,24 @@
 "use strict";
 
-exports.configOverriding = `  Overriding config
+exports.configOverriding = (opts = {}) => {
```

Comment:
You lost two extra spaces at the beginning.

Here we could do this:
```js
exports.configOverriding = ({ cliName = "testplane" } = {}) => `  Overriding config
    To override any config...
```

This way we won't need to have extra `return`, and spacing would be untouched

Reply:
> You lost two extra spaces at the beginning.

thank you!

Fixed.